tools/document-metrics/Cargo.toml

```diff
@@ -14,7 +14,7 @@ clap = { version = "4.1.4", features = ["derive"] }
 serde = { version = "1.0.114", features = ["derive"] }
 serde_json = "1.0.55"
 serde_yaml = "0.9.17"
-syn = "1.0.39"
+syn = { version = "1.0.39", features = ["full"] }
```

Comment:
I'm pretty confused why I had to do this, after merging with master this crate suddenly complained it needed the "full" feature flag, even though it worked without it before